app/components/Downloads/DownloadsList/index.js

```diff
@@ -23,6 +24,23 @@ const DownloadsList = ({
         <Table.Header>
           <Table.Row>
             <Table.HeaderCell>{t('status')}</Table.HeaderCell>
+            <Table.HeaderCell onClick={() => {
+              if (sortArtistAsc){
+                items.sort((a, b) => {
+                  return a.track.artist.name.toLowerCase() > b.track.artist.name.toLowerCase();
+                });
+                setArtistSort(false);
+              } else {
+                items.sort((a, b) => {
+                  return a.track.artist.name.toLowerCase() < b.track.artist.name.toLowerCase();
+                });
+                setArtistSort(true);
+              }
+            }
+            }>{t('Artist')} {
```

Comment:
This, along with `t('Song')` won't do what you want it to do. This is a translation function and the way it works is by getting the strings from appropriate locale by their key. E.g. for english, it looks here: https://github.com/nukeop/nuclear/blob/master/app/locales/en.json

For downloads, it uses the string located under the "downloads" key. To get the artist/title pair for the song in the download queue, you can just access its object directly.